Sito/css/style.css

```diff
@@ -38,6 +38,29 @@ body a:visited {
  *
  */
 
+/* Form */
+form{
```

Comment:
Io creerei una classe perchè se fai così poi magari ci sono dei form che non devono essere fatti così

Reply:
Ho pensato che era la volta buona per uniformarci nei form.
Nella mia parte di CSS ho sovrascritto/aggiunto solo le cose che differivano dal form di "default"
A chi ha già creato i form, in teoria, se ha usato un id non dovrebbe succedere nulla

Reply:
In ogni caso non accettare la pull request così ne discutiamo con tutti e poi decidiamo

Reply:
Eh ok ma se uno vuole il form fatto così ci mette la classe se invece no se la fa da solo..perchè magari qui ci sono regole che uno anche mettendo l'id non modifica e che quindi non sovrascrive e poi gli fa macello

Reply:
Ok, ho creato la classe e l'ho applicata anche alla tua pagina login. Se ha qualche problema dimmi che risolvo il prima possibile